### PR TITLE
feat(server): Multiple ContentTypes

### PR DESCRIPTION
**Pull request by maxbreuker**
- Support for multiple content types
- Conversion of corresponding positions to string arrays
- Adjusted unit tests

### CODE REVIEW

**Review by kryptobi — CHANGES_REQUESTED** [review]
formatting vs. content changes
What are your current changes?

File: MagicBytesValidator/Services/Http/FormFileTypeProvider.cs [hunk 1]
```diff
@@ -45,12 +45,12 @@ public FormFileTypeProvider(Mapping? mapping = null)
...
             }
 
-            if (fileType.MimeType != formFile.ContentType)
+            if (fileType.MimeTypes.Contains(formFile.ContentType) == false)
             {
                 /* This can only occur if the given form file has a file name and its extension indicates a different
                  * MIME type as (also given) Content-Type. This *can* be an indicator that someone is trying to
                  * mess with us. As we are a bit paranoid and also the file type is not unambiguous, we'll throw. */
-                throw new MimeTypeMismatchException(fileType.MimeType, formFile.ContentType);
+                throw new MimeTypeMismatchException(fileType.MimeTypes.First(), formFile.ContentType);
```
Comment (maxbreuker) [inline MagicBytesValidator/Services/Http/FormFileTypeProvider.cs:53]: Use correct content type at exception.

**Review by neon-JS — APPROVED** [review]

File: MagicBytesValidator.Tests/IsValidTests.cs [hunk 2]
```diff
@@ -42,6 +48,18 @@ public async Task Should_validate()
             // Assert
             valid.Should().BeTrue();
         }
+        
+        [Fact]
+        public async Task Should_validate_2()
```
[the remark this answers is not in the document]
Comment (kryptobi) [reply MagicBytesValidator.Tests/IsValidTests.cs:53]: .. really?

[the remark this answers is not in the document]
Comment (kryptobi) [reply MagicBytesValidator.Tests/IsValidTests.cs:53]: @maxbreuker rename ist.. you are not able to merge the pr before.

**Review by kryptobi — COMMENTED** [review]
check comments

**Review by kryptobi — APPROVED** [review]

**Pull request merged**